test/e2e_node/device_plugin_test.go

```diff
@@ -112,13 +110,6 @@ func readDaemonSetV1OrDie(objBytes []byte) *appsv1.DaemonSet {
 func testDevicePlugin(f *framework.Framework, pluginSockDir string) {
 	pluginSockDir = filepath.Join(pluginSockDir) + "/"
 	ginkgo.Context("DevicePlugin", func() {
-		ginkgo.By("Enabling support for Kubelet Plugins Watcher")
-		tempSetCurrentKubeletConfig(f, func(initialConfig *kubeletconfig.KubeletConfiguration) {
-			if initialConfig.FeatureGates == nil {
-				initialConfig.FeatureGates = map[string]bool{}
-			}
-			initialConfig.FeatureGates[string(features.KubeletPodResources)] = true
```

Comment:
Was this feature flag removed? We don't seem to be replacing this with anything.

Reply:
yes, better to move it back until at least the cpumanager feature gate is locked to true